test/test_chain_id_cli_params.py

```diff
@@ -0,0 +1,36 @@
+"""Testing chain id CLI params"""
+
+import subprocess
+
+import pytest
+from starkware.starknet.definitions.general_config import StarknetChainId
+
+from .util import DevnetBackgroundProc, read_stream
+
+ACTIVE_DEVNET = DevnetBackgroundProc()
+
+@pytest.mark.parametrize(
+    "chain_id",
+    [member.name for member in StarknetChainId],
+)
+def test_chain_id_valid(chain_id):
+    """Test if chain id works"""
+    proc = ACTIVE_DEVNET.start(
+        "--chain-id",
+        chain_id,
+    )
+    assert proc.returncode is None
+
+def test_chain_id_invalid():
+    """Test if the invalid chain id fails"""
+    proc = ACTIVE_DEVNET.start(
+        "--chain-id",
+        "",
+        stderr=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+    )
+    assert (
+        f"The value of --chain_id must be in {[member.name for member in StarknetChainId]}, got:"
+        in read_stream(proc.stderr)
+    )
+    assert proc.returncode == 1
```

Comment:
A test should also be included which does an actual invoke with a chain id specified (all those functions we use for generating an invoke/signature would need to be updated with a default value of `StarknetChainId.TESTNET`).

Reply:
I added 2 new test with deploy and invoke
https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/28a305c55607f8eae65177ba2ad8aa0e57c19888